recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/images/core-image-weston.bbappend

```diff
@@ -40,5 +40,15 @@ populate_vmlinux () {
     find ${STAGING_KERNEL_BUILDDIR} -iname "vmlinux*" -exec mv {} ${DEPLOY_DIR_IMAGE} \; || true
 }
 
-IMAGE_POSTPROCESS_COMMAND += "populate_vmlinux; "
+prepare_graphic () {
```

Comment:
I would like to see this in a separate recipe, which has dependencies on other three ones. 
Having separate recipe will allow you to package DDK without rebuilding whole image.